pages/common/fls.md

```diff
@@ -0,0 +1,16 @@
+# fls
+
+> List files and directories in an image file or device.
+> More information at https://wiki.sleuthkit.org/index.php?title=Fls.
+
+- Built a recursive fls list over a device:
+
+`fls -r -m {{Original_Path}} {{/dev/examplepartition}}`
+
+- The above, for a partition starting at a specific sector using a pre-defined timezone:
```

Comment:
I'm not sure should we keep `The above` here.

Reply:
thanks